router/server.go

```diff
@@ -140,5 +128,18 @@ func main() {
 	if err != nil {
 		shutdown.Fatal(err)
 	}
+
```

Comment:
There's another `ListenAndServe` a few lines above the fold in a goroutine, should that be split to ensure bind before discoverd annouce too?